assets/terraform-modules/aws/flatcar-linux/kubernetes/workers/workers.tf

```
@@ -52,6 +52,68 @@ resource "aws_autoscaling_group" "workers" {
   ])
 }
 
+# IAM Policy.
+resource "aws_iam_instance_profile" "csi-driver" {
```

Resource names should use underscores.
```suggestion
resource "aws_iam_instance_profile" "csi_driver" {
```